polkadot/runtime/common/src/integration_tests.rs

```diff
@@ -211,21 +215,32 @@ impl paras::Config for Test {
 	type UnsignedPriority = ParasUnsignedPriority;
 	type QueueFootprinter = ();
 	type NextSessionRotation = crate::mock::TestNextSessionRotation;
+	type OnCodeUpgrade = Registrar;
 	type OnNewHead = ();
 }
 
 parameter_types! {
 	pub const ParaDeposit: Balance = 500;
 	pub const DataDepositPerByte: Balance = 1;
+	pub const UpgradeFee: Balance = 2;
+	pub const RelayNetwork: NetworkId = NetworkId::Kusama;
 }
 
+pub type LocationToAccountId = (
```

Comment:
We should be using the newest things in mocks if we can.

This is the new location converter that does the same as previous ones but in an easier to standardize and more extensible way:

```rust
pub type LocationToAccountId = HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>;
```